wbooks/src/app/components/BookItem/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { View, Image, Text } from 'react-native';
+import { Book } from '@interfaces//books';
+
+import styles from './styles';
+
+function BookItem({ author, imageUrl, title }: Book) {
+  return (
+    <View style={styles.cardContainer}>
+      <View style={styles.imageContainer}>
+        <Image source={{ uri: imageUrl }} />
+      </View>
```

Comment:
That View is unnecessary, Image can receive a style
```suggestion
     <Image source={{ uri: imageUrl }} style={styles.image} />
```